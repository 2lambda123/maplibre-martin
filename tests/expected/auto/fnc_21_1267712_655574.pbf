(
public.function_source	"	îè0(€ x