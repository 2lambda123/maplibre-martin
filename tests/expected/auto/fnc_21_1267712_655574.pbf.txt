=============================================================
layer: 0
  name: public.function_source
  version: 2
  extent: 4096
  feature: 0
    id: (none)
    geomtype: point
    geometry:
      POINT(823,3124)
    properties:
